src/less/day-slot.less

```diff
@@ -30,20 +31,29 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
+
+  .rbc-timeslice-group {
+    padding: 0;
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
```

Comment:
this was where the 40px 40px comes from.  rbc-time-slot was 20px, and we want the height of the group to equal 2 of these.  I agree min-height and height is a weird redundancy, but it came from here.  No problem correcting it if you want, just moving to height